cmake/StaticAnalyzers.cmake

```diff
@@ -7,3 +7,17 @@ if(TW_ENABLE_CLANG_TIDY)
         message(SEND_ERROR "Clang-Tidy requested but executable not found.")
     endif()
 endif()
+
+if (TW_ENABLE_PVS_STUDIO)
+    macro(tw_add_pvs_studio_target target)
+        message(STATUS "PVS-Studio analyzer enabled - ${CMAKE_SOURCE_DIR}/tools/pvs-studio/config.cfg")
+        include(cmake/PVS-Studio.cmake)
+        pvs_studio_add_target(TARGET TrustWalletCore.analyze ALL
+                OUTPUT FORMAT json
+                ANALYZE ${target}
+                MODE GA:1,2
+                LOG target.err
+                CONFIG ${CMAKE_SOURCE_DIR}/tools/pvs-studio/config.cfg
+                )
```

Comment:
Very naive usage for now, more options here: https://pvs-studio.com/en/docs/manual/6591/

Do we want to have extra options for IDE, saving the file instead of dumping it, do we want more severity warnings etc - open to discuss it here.

Reply:
How long will it take to finish scanning?

Reply:
> How long will it take to finish scanning?

4-5 minutes